src/main.js

```diff
@@ -50,10 +57,14 @@ function generateRandomPoster() {
 }
 
 function createRandomPoster() {
-  var randomPoster = generateRandomPoster();
-  posterImage.src = randomPoster.imageURL;
-  posterTitle.innerHTML = randomPoster.title;
-  posterQuote.innerHTML = randomPoster.quote;
+  // var randomPoster = generateRandomPoster();
+  // posterImage.src = randomPoster.imageURL;
+  // posterTitle.innerHTML = randomPoster.title;
+  // posterQuote.innerHTML = randomPoster.quote;
```

Comment:
When pushing your code to be merged, try to remove as many lines of commented code as possible. Unless they are specifically inside of the code base for direction for yourself or other developers.